torchtext/experimental/vocab.py

```diff
@@ -0,0 +1,97 @@
+import logging
+
+import torch
+import torch.nn as nn
+
+logger = logging.getLogger(__name__)
+
+
+class Vocab(nn.Module):
+    r"""Creates a vocab object which maps tokens to indices.
+
+    Arguments:
+        ordered_dict (collections.OrderedDict): object holding the frequencies of each token found in the data.
+        min_freq: The minimum frequency needed to include a token in the vocabulary.
+            Values less than 1 will be set to 1. Default: 1.
+        specials: The tuple of special tokens (e.g., padding or eos) that will be prepended to the vocabulary.
+            The first value should always be the unknown token Default: ['<unk'>, '<pad>']
```

Comment:
This line is not clear. The specials will be prepended to the vocabulary list so the order of the special tokens in `specials` matters, right? And what happen if `<unk>' is not in `specials`? What happen if users assign a custom unknown token?

Reply:
Yes I can specify that the ordering of the specials token matter. The comment about the unk token is incorrect. I have a seperate `unk_token` parameter that the user needs to pass in. I will update the comment to reflect this!